assets/dev/js/editor/document/hooks/data/document/elements/helper.js

```diff
@@ -11,6 +11,15 @@ export default class Helper {
 					elements: [],
 				};
 
+				const component = $e.components.get( 'editor/documents' ),
+					command = 'editor/documents/elements',
+					query = {
+						documentId: elementor.documents.getCurrent().id,
+						elementId: model.id,
+					};
+
+				$e.data.setCache( component, command, query, model );
```

Comment:
```suggestion
				// Since, createSectionColumns called from 'create-section-columns' hook. and not using command.
				$e.data.setCache( component, command, query, model );
```